src/App.js

```diff
@@ -0,0 +1,127 @@
+import TodoList from './modules/todo-list/TodoList';
+import SignUp from './modules/SignUp';
+import SignIn from './modules/SignIn';
+import SignUpConfirmation from './modules/SignUpConfirmation';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+
+
+function App() {
+    const [newToDoItem, setNewToDoItem] = useState('');
```

Comment:
please movo todo related logic to todo list component

Reply:
have you pushed those changes?

Reply:
No, not yet, I will push all changes together